rust_src/src/process.rs

```diff
@@ -1,7 +1,7 @@
 //! Functions operating on process.
 
 use remacs_macros::lisp_fn;
-use remacs_sys::{Fmapcar, Lisp_Process, Qcdr, Qlistp, Vprocess_alist};
+use remacs_sys::{fget_pid, EmacsInt, Fmapcar, Lisp_Process, Qcdr, Qlistp, Vprocess_alist};
```

Comment:
Please put `fget_pid` on its own line. We are breaking up the big imports to reduce conflicts.